7.redundancy_analysis/notebooks/0.redundancy_analysis.ipynb

```diff
@@ -0,0 +1,1098 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    K = min(N, P, Q)
</pre><p>Consider checking if N &lt; min(P, Q) first</p><p> </p><p>Why do K depend on the number of rows N?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>I do not think I should need to check as if N is indeed &lt; the min(P,Q) then it will be picked up in the expression min(N,P,Q) but if it is larger then it wont be used.</p><p> </p><p>In terms of why - this is a great questions that the answer is not documented and I added documentation to address this. Thank you for checking in on this!</p>